alacritty_terminal/src/term/search.rs

```diff
@@ -291,6 +286,13 @@ impl<T> Term<T> {
         match direction {
             Direction::Right if cell.flags.contains(Flags::WIDE_CHAR) => {
                 iter.next();
+
+                // If iterates `iter` when finds a wide character without wide character spacer,
+                // one character is skipped unintentionally. So returns `iter` to one previous
+                // position.
+                if !iter.cell().flags.contains(Flags::WIDE_CHAR_SPACER) {
+                    iter.prev();
+                }
```

Comment:
Couldn't we just not call `iter.next()` in this case? Calling next just to call prev immediately seems like a strange process.